ui/components/ui/nickname-popover/nickname-popover.component.js

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { text } from '@storybook/addon-knobs';
+import { I18nContext } from '../../../contexts/i18n';
+import Popover from '../popover';
+import Button from '../button';
+import Identicon from '../identicon/identicon.component';
+import { shortenAddress } from '../../../helpers/utils/util';
+import CopyIcon from '../icon/copy-icon.component';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+
+const NicknamePopover = ({ address, onClose = null, onAdd = null }) => {
+  const t = useContext(I18nContext);
+
+  const onAddClick = useCallback(() => {
+    onAdd(address);
+    onClose();
+  }, [address, onClose, onAdd]);
+
+  const [copied, handleCopy] = useCopyToClipboard();
+
+  return (
+    <div className="nickname-popover">
+      <Popover onClose={onClose} className="nickname-popover__popover-wrap">
+        <Identicon
+          address={text(
+            'Address',
+            '0x5e6DaAD1BE117e26590F9eEcD509336ABFBe5966',
```

Comment:
Should this value be hardcoded?

Reply:
@darkwing It shouldn't be hardcoded address, my mistake, I fix that. Thank you for catching this.
You can see changes here: 29a0f1c9536439d5c5b7469b4115f3d73b74cc05
Is it okay now?